src/openrct2/platform/posix.c

```diff
@@ -391,6 +391,17 @@ sint32 platform_enumerate_files_begin(const utf8 *pattern)
 				safe_strcpy(paths[idx], dir_name, path_len);
 				safe_strcat_path(paths[idx], d->d_name, path_len);
 				log_verbose("paths[%d] = %s", idx, paths[idx]);
+
+				// macOS uses decomposed Unicode strings (e.g. an 'e' and a combining accent) in filenames
+				// This causes problems with the sprite font, as the font only contains precomposed characters
+				// The block below converts all filename strings to their precomposed form, preventing mojibake
+				#ifdef __MACOSX__
```

Comment:
Ah, okay - I was following precedent elsewhere in the file (L813) - I'll fix both.